atom/app/atom_main_delegate.cc

```diff
@@ -156,6 +156,20 @@ base::FilePath InitializeUserDataDir() {
     command_line->AppendSwitchPath(switches::kUserDataDir, user_data_dir);
   }
 
+#if defined(OS_WIN)
```

Comment:
@bridiver can you expand on that more?

Are you saying that the change needs to do something like the following?
```
command_line->AppendSwitchPath(switches::kUserDataDirName, user_data_dir);
```

Why would we do this?  The change as-is looks OK to me